python/composio/__init__.py

```diff
@@ -1,7 +1,23 @@
-import atexit
+# pylint: disable=wrong-import-position
 
-from composio.client import Composio
-from composio.client.enums import (
+import sentry_sdk
+
+
+sentry_sdk.init(
+    dsn=(
+        "https://071d95df77abfdaaa87cd9441f42e12f@o4507825828462592.ingest.de."
+        "sentry.io/4507833628557392"
```

Comment:
Just checking this is public stuff right?

Reply:
yes